File: android/DFM/src/main/java/com/dontflymoney/watchers/DescriptionWatcher.java
package com.dontflymoney.watchers;

import com.dontflymoney.entities.Move;
import com.dontflymoney.viewhelper.AfterTextWatcher;

public class DescriptionWatcher extends AfterTextWatcher
{
	private Move move;

	public DescriptionWatcher(Move move)
	{
		this.move = move;
	}

	@Override
	public void textChanged(String text)
	{
		move.Description = text;
	}
}

File: android/DFM/src/main/kotlin/com/darakeon/uiHelpers/watchers/DescriptionWatcher.kt
package com.dontflymoney.watchers

import com.dontflymoney.entities.Move
import com.dontflymoney.viewhelper.AfterTextWatcher

class DescriptionWatcher(private val move: Move) : AfterTextWatcher() {

    override fun textChanged(text: String) {
        move.Description = text
    }
}
